examples/js/src/store_and_hash_client.js

```diff
@@ -0,0 +1,34 @@
+const CommonMessages = require('./common_pb.js');
+const { loggingStreamHandler } = require('./util.js');
+const StoreAndHashMessages = require('./storeandhash-service_pb.js');
+const StoreAndHashService = require('./storeandhash-service_grpc_pb.js');
+const grpc = require('@grpc/grpc-js');
+
+const storeAndHashSvc = new StoreAndHashService.StoreAndHashServiceClient(
+    'psg-testnet.iog.services:2001',
+    grpc.credentials.createSsl()
+);
+
+const reqs = [
+    new StoreAndHashMessages.StoreAndHashRequest()
+        .setDetails(
+            new StoreAndHashMessages.UploadDetails()
+                .setAws(new StoreAndHashMessages.AwsCredentials()
+                    .setKeyid("AWS_ACCCESS_KEY")
```

Comment:
Same comment as above ... I'm just not sure if I should replace these values or how they get into the code....

Reply:
Fixed too.